domain-sample-data/build.gradle

```diff
@@ -8,4 +8,8 @@ repositories {
 
 dependencies {
   compile project(':domain-model')
+  compile project(':ccd-adapter')
+  compile group: 'uk.gov.hmcts.reform', name: 'core-case-data-store-client', version: '4.6.2'
```

Comment:
I don't think this dependency is needed here. is it?

Reply:
done.